packages/frames.js/package.json

```diff
@@ -209,12 +209,22 @@
     },
     "./next/pages-router": {
```

Comment:
maybe we should rename this entrypoint to `next/pages-router/server` to distinguish properly?